android-auto-app/README.md

```diff
@@ -0,0 +1,20 @@
+## Welcome to the Mapbox Android Auto demo
+
+This app is showcases the `:android-auto` module which is is integrated with
+a Mapbox internally tested app, where we test it on real cars. We will periodically
```

Comment:
`Mapbox internally tested app` here I guess means 1Tap. Now that we are porting all the capabilities over from 1Tap, instead of mentioning `internally tested app` can we refer to these set of [examples](https://github.com/mapbox/mapbox-navigation-android-examples/tree/main/android-auto/src/main/java/com/mapbox/examples/androidauto/car)

Reply:
The idea here was to mention that the solution is being tested in real cars. I found it necessary to mention it "internal" because no one can access it through Google Play. This readme is not ported from or to 1tap. 

That said, this section is providing an introduction for what this project is, how it is tested. The audience is less for mapbox and more for outsiders.

I still think we should mention this is tested in real cars. But maybe the wording can be different?

Reply:
Although, thanks for adding the main page readme. Rethinking what we want to put here 🤔

Reply:
> I still think we should mention this is tested in real cars.

Oh definitely we should mention this is tested in real cars, but can't we run [examples](https://github.com/mapbox/mapbox-navigation-android-examples/tree/main/android-auto/src/main/java/com/mapbox/examples/androidauto/car) in real cars? Maybe it's not 100% ported from 1Tap and has only 50% of what 1Tap can do. However eventually it will be complete.

Reply:
After integrating with the main README.md, decided to completely change this. Resolving this thread and we can discuss another iteration